cfme/services/myservice/ui.py

```diff
@@ -369,7 +369,7 @@ def step(self, *args, **kwargs):
         self.prerequisite_view.navigation.select('Services', 'My Services')
 
     def resetter(self, *args, **kwargs):
-        self.prerequisite_view.navigation.select('Services', 'My Services')
+        self.view.myservice.tree.click_path('Active Services')
```

Comment:
I suggest to fix the problem like that:
```suggestion
        self.view.navigation.select('Services', 'My Services')
```
Because otherwise the `cfme/tests/services/test_service_performance.py::test_services_performance` will fail as the services tree will not get displayed in FF.

Reply:
@jadhaj if you check steps in navigation its only doing that work `select('Services', 'My Services')` then why we need to do same thing in `resetter`?

secondly, we are testing these thing in chrome not in firefox. Yup, I know most of navigation and widgets not working properly on firefox.

Reply:
@jaryn interesting observation, are you sure this isn't a BZ in CF?

Going through the navigation tree really is a re-navigation, not a resetter action.  If the test needs specific navigation behavior, that should be handled in the test.

"Normal" navigation reset behavior, I think, would just be to click the accordion tree, not completely re-navigate.